orttraining/orttraining/python/training/_ortmodule_io.py

```diff
@@ -5,6 +5,55 @@
 import torch
 import warnings
 
+
+class _InputInfo(object):
+    def __init__(self, names, shape, require_grad_names = [], dynamic_axes = {}, schema = []):
+        self.names = names
+        self.shape = shape
+        self.require_grad_names = require_grad_names
+        self.dynamic_axes = dynamic_axes
+        self.schema = schema
+
+    def id(self) -> int:
+        return hash(repr(self.schema))
```

Comment:
Is this used anywhere? I don't think it's overriding the behavior of `id()`, if that's what you're intending....

Perhaps you want to override `__hash__()` and/or `__eq__()`?

Reply:
This is a leftover. Removing it!